src/charts/helpers/auto-format.js

```diff
@@ -28,7 +30,11 @@ function applyNumberFormatWithSuffix(tokens, value) {
     prefixTranslation[tokens[1]]
   ) {
     try {
-      return d3.formatPrefix(tokens[0], prefixTranslation[tokens[1]])(value)
+      let formatted = d3.formatPrefix(tokens[0], prefixTranslation[tokens[1]])(value)
```

Comment:
Is there a reason why we can't just do a `.toUpperCase(tokens[1])` when we access the properties on `prefixTranslations`?  That way we could remove the duplicate values on `prefixTranslations`, and also simplify the cyclomatic complexity of the conditional below, since we won't have to be dealing w/ the capitalization, or not, of the given token....